test/RootStateSchema.json

```diff
@@ -903,6 +903,48 @@
             ],
             "type": "object"
         },
+        "CoreTypes.Metadata_1": {
```

Comment:
This seems suspect, do we have a duplicated state variable?

Reply:
this is not a duplicated state variable - the extra `_1` objects are type variables only.

this happened because there were duplicated walletconnect dependencies in the yarn.lock. i've resolved this